config.go

```diff
@@ -140,6 +140,10 @@ type bitcoindConfig struct {
 	ZMQPath string `long:"zmqpath" description:"The path to the ZMQ socket providing at least raw blocks. Raw transactions can be handled as well."`
 }
 
+type extpreimageConfig struct {
```

Comment:
this should be camelCase. Either `externalPreimageConfig` or `extPreimageConfig`

Reply:
Package names in golang are all lowercase, all one word. The package in this case is `extpreimage`. This is the configuration for that package, which is why the casing appears like it does. If I do something like `extPreimageConfig`, it's the only place where the 'P' is capitalized.

Reply:
this is a struct name, tho. I get that we want them to match. To the untrained eye, this looks like a typo.